### DISCUSSION TITLE
How do I get my custom SQL query relationships to appear?

### DISCUSSION THREAD

**Discussion by EvanGllt** [Budibase/budibase#14170] [Help]
Hello, there! 👋

I want to be able to see the relationships I've defined in my table, knowing that I'm using a custom SQL query:
![aide 4](https://github.com/user-attachments/assets/32670c9b-c2ad-42a8-b532-1e370a6bec0c)

Except I can't find a way to make them appear with a custom query 😮‍💨

I'm connected to an MS SQL SERVER database.

Can you help me? Thanks in advance.

**Comment by benaunt** [comment]
I think you just manually re-define them utilising the existing relationship columns from SQL within the Budibase relationship builder.

**Reply by EvanGllt** [reply]
Hi and thanks for your reply.

In fact I have defined relationships in budibase : 
![1](https://github.com/user-attachments/assets/6289f7cc-385c-471e-89ba-9d058f703a94)
![3](https://github.com/user-attachments/assets/0a6fb06b-0b3f-48c6-b14e-c36bbc280dea)
![4](https://github.com/user-attachments/assets/8a18bcdc-9a48-4837-ac07-bf6da7655ec3)
![5](https://github.com/user-attachments/assets/9f68e094-4c31-4f1a-976e-243a7891a3c4)

And my SQL query :
![2](https://github.com/user-attachments/assets/d4dba10c-1f7d-45cb-8040-561117cb8a81)

I don't know if I'm doing it wrong but I can think of other ways of doing it...
